.storybook/preview.js

```diff
@@ -163,6 +164,20 @@ export const decorators = [
               color: 'rgb(0, 164, 124)',
             },
           },
+          LinePositive: {
```

Comment:
Let's pass these colors along side the `DataSeries` instead of setting a new theme for each.